todo_app/data/view_model.py

```diff
@@ -0,0 +1,32 @@
+from todo_app.data.trello_items import Item
+
+class ViewModel:
+    def __init__(self, items : list[Item]):
+        self._items = items
+
+    @property
+    def done_items(self) -> list[Item]:
+        return_li = []
+        for item in self._items:
+            if item.status =='Done':
+                return_li.append(item)
+        return return_li
```

Comment:
Nice, you've got a clean effective approach here! Not a change I necessarily recommend, but worth comparing with the more concise (but arguably less clear) list-comprehension approach:
```python
    return [item for item in self._items if item.status == 'Done']
```